app/src/main/java/ch/epfl/favo/user/UserUtil.java

```diff
@@ -39,7 +41,7 @@ public static UserUtil getSingleInstance() {
    * @param user A user object.
    * @throws RuntimeException Unable to post to DB.
    */
-  public void postAccount(User user) throws RuntimeException {
+  public void postUser(User user) throws RuntimeException {
```

Comment:
I think this should return a competable future that can be checked for completion in the UI

Reply:
The add calls are void from the database level since you cant return a future for anything other than a 'get' call. It throws a runtimeException if it has not been completed successfully which we can perhaps throw to the frontend so that it can check for completion?